src/commands/Minion/tob.ts

```diff
@@ -352,20 +352,30 @@ export default class extends BotCommand {
 		return msg.channel.send(str);
 	}
 
-	async run(msg: KlasaMessage) {
+	async run(msg: KlasaMessage, [input]: [string | undefined]) {
 		const hardKC = await msg.author.getMinigameScore('tob_hard');
 		const kc = await msg.author.getMinigameScore('tob');
 		const attempts = await msg.author.settings.get(UserSettings.Stats.TobAttempts);
 		const hardAttempts = await msg.author.settings.get(UserSettings.Stats.TobHardModeAttempts);
 		const gear = calculateTOBUserGearPercents(msg.author);
 		const deathChances = calculateTOBDeaths(kc, hardKC, attempts, hardAttempts, false, gear);
 		const hardDeathChances = calculateTOBDeaths(kc, hardKC, attempts, hardAttempts, true, gear);
-
+		if (input === 'gear') {
+			return msg.channel.send(`**Melee Gear Score:** <:Elder_maul:403018312247803906> ${gear.melee.toFixed(1)}%
+**Range Gear Score:** <:Twisted_bow:403018312402862081> ${gear.range.toFixed(1)}%
+**Mage Gear Score:** <:Kodai_insignia:403018312264712193> ${gear.mage.toFixed(1)}%
+**Total Gear score:** ${gear.total.toFixed(1)}%\n
+Death Chances: ${deathChances.deathChances.map(i => `${i.name}[${i.deathChance.toFixed(2)}%]`).join(' ')}
+Hard Mode Death Chances: ${hardDeathChances.deathChances
+				.map(i => `${i.name}[${i.deathChance.toFixed(2)}%]`)
```

Comment:
Just show the gear scores in the normal tob message. These death chances are duplicated here too.